src/hotspot/share/cds/cdsConfig.hpp

```diff
@@ -46,6 +46,7 @@ class CDSConfig : public AllStatic {
   static char* _dynamic_archive_path;
 
   static bool  _old_cds_flags_used;
+  static bool  _disable_heap_dumping;
```

Comment:
Following with the previous implementation, I'd prefer `can_dump_heap` to avoid the double negative.

Reply:
`_disable_heap_dumping` is for unconditionally disabling heap dumping. The opposite does not mean we can dump the heap -- heap dumping can also be disabled if ZGC is used. So `can_dump_heap` will be misleading.